traffic-participants.rst

```diff
@@ -0,0 +1,96 @@
+Modeling traffic participants
+==============================
+
+Traffic participants
+--------------------
+
+A traffic participant is an element of the simulated world, which can change its position and orientation during simulation time.
+A traffic participant represents one of the following:
+
+- A living being.
+- A means of transportation for living beings
+- A means of transportation for goods
+- Any other movable object that may travel the road network
+
+Pedestrians and animals are examples of traffic participants that are living beings.
+Vehicles are examples of traffic participants that are means of transportation.
+Therefore, the ego vehicle is also a traffic participant.
+
+The following figure shows the interface of a traffic participant.
+
+.. image:: _static/images/osi-traffic-participant-principle.png
+   :align: center
+   :alt: Traffic participant interfaces
+
+With every simulation step, an OSI traffic participant model receives ground truth data from the environment around itself, the sensor view.
+A traffic participant can output its own perceivable state, the traffic update.
+Traffic commands influence the behavior of the traffic participant model.
+Traffic commands allow event-based communication towards the traffic participant, for example, at certain simulation steps.
+OSI only specifies the interface of the traffic participant.
+OSI is agnostic to the architecture of the simulation environment and the traffic participant.
+
+
+Use cases for traffic participants
+----------------------------------
+
+Different models are involved in modeling a traffic participant.
```

Comment:
Different models can be involved ...

Reply:
I intentionally avoided using a modal verb because they have very specific meaning in the ASAM context.

If we want to use a modal verb, we should decide which one to use.
In ASAM parlance, "can" denotes something that is technically possible while "may" denotes something that is expressly allowed within the limits of the standard.

Reply:
then I think "may be involved" is correct.

Reply:
- [ ] Change to "may be involved".